Team417_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team417_CENTERSTAGE/baseprograms/BaseAutonomous.java

```diff
@@ -187,52 +427,80 @@ PoseAndAction getDriveAction(boolean isRed, boolean isFar, SpikeMarks location,
         }
 
         // in MeepMeep, intake needs to be null however .stopAndAdd() can't be null because it will crash so we set to a random sleep
-        if(intake == null) {
+        if (intake == null) {
             intake = new SleepAction(3);
         }
 
-        TrajectoryActionBuilder spikeLeft = this.drive.actionBuilder(xForm(new Pose2d(-34, -60, Math.toRadians(90))));
-        spikeLeft = spikeLeft.splineTo(xForm(new Vector2d(-34, -36)), xForm(Math.toRadians(90)))
-                .splineTo(xForm(new Vector2d(-38, -34)), xForm(Math.toRadians(180) + (1e-6)))
+        TrajectoryActionBuilder spikeLeft = this.drive.actionBuilder(xForm(new Pose2d(-34, -64, Math.toRadians(90))));
+        spikeLeft = spikeLeft.splineTo(xForm(new Vector2d(-34, -37)), xForm(Math.toRadians(90)))
+                .splineTo(xForm(new Vector2d(-33.5, -34)), xForm((Math.toRadians(180))))
```

Comment:
The before version added `1e-6` to `Math.toRadians(180)`. That was to ensure that it turned in the correct direction. Why did that code get deleted? Was it because it wasn't commented in the old code?